Taskfile.Shared.yml

```diff
@@ -53,7 +53,7 @@ tasks:
     label: "build-{{.NAME}}"
     dir: coprocessors/{{.NAME}}
     cmds:
-      - docker build . -t {{.NAME}}-co
+      - DOCKER_BUILDKIT=1 docker build . -t {{.NAME}}-co
```

Comment:
I was looking into this some more, and I'm not sure docker packages are all up to 23 for linux, yet, so maybe this would be good to include in all our `docker build` commands (coprocessors, subgraphs, and loadtester)?

Reply:
Yeah, I needed it to run on GitHub Actions. It's possible I could update docker somehow instead? But I figure just including the env var would make it the most flexible.